src/coreclr/nativeaot/Runtime/riscv64/UniversalTransition.S

```diff
@@ -136,6 +136,11 @@
         mv a1, t1                                         // Second parameter to target function
         jalr t0, t1, 0                                    // Jump to the function in t1
 
+        // We cannot make the label public as that tricks DIA stackwalker into thinking
```

Comment:
This comment should be deleted here and everywhere else.

"DIA stackwalker" is Windows specific and the auxiliary variable hack that this comment talks about was deleted in https://github.com/dotnet/runtime/pull/97853

Reply:
Do we need ALTERNATE_ENTRY? Aliases are still in use in code e.g. `ReturnFromUniversalTransition`, which was giving the missing symbol error at link time.

Reply:
Yes, the `ALTERNATE_ENTRY` is still needed.